### PR TITLE
Use an action with an actual type to get the initialSate

### PR DESCRIPTION
**Pull request by yched**
Fixes https://github.com/erikras/multireducer/issues/114

### CODE REVIEW

File: src/plainMultireducer.js [hunk 1]
```diff
@@ -11,9 +11,10 @@ export default function plainMultireducer(reducers, reducerKey) {
     }
   }
 
+  const initAction = {type: '@@multireducer/INIT'};
```
Comment (yesmeck) [inline src/plainMultireducer.js]: Need a space after "{" and "}".

Comment (yched) [inline src/plainMultireducer.js]: Oops, sure

**Comment by yched** [comment]
Right, forgot immutableMultireducer. Done

**Comment by yesmeck** [comment]
Published in v3.0.1.

**Pull request merged**